sites/partners/src/LinkComponent.tsx

```diff
@@ -0,0 +1,16 @@
+import { PropsWithChildren } from "react"
```

Comment:
This is identical to the one in public, so if we go the route of merging this into @jaredcwhite public, should this be moved into ui-components so they can use the same one?

Reply:
I don't like the duplication either, but the reason for this at all is the line `import Link from "next/link"`

Through discussions with SFDS, we were tasked to ensure ui-components doesn't have any dependency on Next (as in it we literally remove it from its `package.json`), so that's why we wrap Link here and push it down the stack via a context. This does bring to mind something the team discussed quite a long time ago: that maybe we need a shared package for bits of Next-specific view/helper code we need in both public+partners that is a separate thing entirely from ui-components, but at the time we decided that extra complexity wasn't worth it.

Reply:
Yup, exactly as @jaredcwhite said that's the reason why I duplicated this component.
I also agree we should think about a shared package for both websites.